json-rpc/src/fuzzing.rs

```diff
@@ -43,8 +43,16 @@ pub fn fuzzer(data: &[u8]) {
     let url = format!("http://{}", address);
     let json_request;
     match serde_json::from_slice::<serde_json::Value>(data) {
-        Err(_) => return, // should not throw error or panic on invalid fuzzer inputs
-        Ok(request) => json_request = request,
+        Err(_) => {
+            // should not throw error or panic on invalid fuzzer inputs
+            if cfg!(test) {
+                panic!();
+            }
+            return;
```

Comment:
I bet in C you could do return cfg!(test) && panic!()
I don't think rust would ever allow that lol...